app/services/event_handlers.py

```diff
@@ -121,12 +122,14 @@ async def transform_and_route_observation(observation):
                 pubsub_message = build_gcp_pubsub_message(
                     payload=transformer_event.dict(exclude_none=True)
                 )
+                # Set ordering key only for updates
+                ordering_key = str(observation.gundi_id) if observation.observation_type == StreamPrefixEnum.event_update.value else ""
```

Comment:
@marianobrc does this mean the original event will have `ordering_key=''`?

Reply:
yes, an empty string as an ordering key is the default in the GCP SDK when no ordering key is set. The objective here is to set an ordering key only for updates, and not for every event or observation, as it has an impact on latency. The order between the original event and the update isn't important as that's handled later in the dispatcher.

Reply:
> The order between the original event and the update isn't important as that's handled later in the dispatcher.

Got it. Thanks!